### ISSUE TITLE
Staging Review finding: Alert link is not keyboard focusable

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128402]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool Team 1
**Product:** Claim Status Tool
**Feature:** Document Submission Status

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

For the "We need you to submit files by mail or in person" alert, the link "Learn how to submit documents by mail or in person" link is only focusable when the alert has received programmatic focus. When the user is tabbing around the page normally, the link can't receive focus.  I believe the issue is that the link is inheriting the `"tabindex="-1"` that is on the parent alert, so it cannot receive focus unless the parent has focus. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="456" height="593" alt="link inherits tabindex" src="https://github.com/user-attachments/assets/8c6562bc-5098-4031-a896-56fa876bb506" />

</details>

## Recommended action

Rather than put the focus/`tabindex="-1" on the whole alert, put it on the heading of the alert. 

Additionally, because this is a dynamic alert, it should be announced via an `aria-live` region.
https://design.va.gov/components/alert/#assign-an-appropriate-aria-role

Especially in this scenario, as the screen reader announcements are in a queue, so the screen reader is announcing the dialog and the upload progress updates even once the dialog closes, so the user isn't aware that there is an alert on screen for several seconds. Using an aria-live region to proactively update the user is appropriate in this scenario.  

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by dfitchett** [comment]
Issue has been resolved:
https://github.com/user-attachments/assets/7fb10e85-2172-4f4f-a308-3c0e92f013bb

**Comment by amylai-va** [comment]
PO review completed ✅
